lib/suma/admin_api/commerce_offerings.rb

```diff
@@ -80,14 +81,15 @@ class PicklistEntity < BaseEntity
     Suma::AdminAPI::CommonEndpoints.create(
       self,
       Suma::Commerce::Offering,
-      DetailedEntity,
+      DetailedOfferingEntity,
     ) do
       params do
         requires :image, type: File
         requires(:description, type: JSON) { use :translated_text }
-        requires(:fulfillment_prompt, type: JSON) { use :translated_text }
-        requires(:fulfillment_confirmation, type: JSON) { use :translated_text }
-        requires :fulfillment_options,
+        optional(:fulfillment_prompt, type: JSON) { use :translated_text, allow_blank: true  }
```

Comment:
Why do we only want allow_blank: true here and not everywhere these params are used?

Reply:
@rgalanakis In some cases, I think there shouldn't be empty :translated_text such as payment triggers `receiving_ledger_contribution_text` param. Ultimately we could set all to allow_blank but I'm not sure if thats a good idea.